analyses/cnv-frequencies/README.md

```diff
@@ -63,7 +63,7 @@ Python functions to create copy number variation (CNV) cancer type and study gen
 
 Usage:
 ```bash
-python3 analysis/cnv-frequencies/01-cnv-frequencies.py HISTOLOGY_FILE CNV_FILE PRIMARY_TUMORS RELAPSE_TUMORS
+python3 analysis/cnv-frequencies/01-cnv-frequencies.py HISTOLOGY_FILE CNV_FILE  AC_PRIMARY_TUMORS AC_RELAPSE_TUMORS EC_PRIMARY_TUMORS EC_RELAPSE_TUMORS
```

Comment:
> For the usage for `.sh` on line 57, since we assumed this would be run in the root dir, maybe we should change this to:
> `bash analyses/cnv-frequencies/run-cnv-frequencies-analysis.sh`

It does not matter where you run the script. There are no hardcoded input file paths in the script.  The bash script calls the python script with inputs parsed via command-line parameters.  It won't break no matter where you run it from as long as you correctly specify inputs on the command-line.

Reply:
Hmm - when I ran in from the analyses directory, it gave the following error:
```
root@12419ebafce3:/home/rstudio/OpenPedCan/analyses/cnv-frequencies# bash run-cnv-frequencies-analysis.sh 
python3: can't open file 'analyses/cnv-frequencies/01-cnv-frequencies.py': [Errno 2] No such file or directory
```
And I think the input params in the bash script are [hard-coded with the assumption that the data files are from `data/*`](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/9b99375b8a826f75038e77da601ad52f5f4f171e/analyses/cnv-frequencies/run-cnv-frequencies-analysis.sh#L16-L20) ?